.github/config.yml

```diff
@@ -17,8 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/Layer5-celebration.png)
-
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="img\readme\meshery-text-dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="img\readme\meshery-text-light.svg">
+  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
+ </picture>
```

Comment:
- Wrong source, and we need Raw github URLs as sources
- Also, the images should be in the welcome directory replacing the layer5-celebration.png
- We recommend using uniform name conventions for images throughout our repos, Meshery-celebration-Dark.svg and Meshery-celebration-Light.svg

There are similar PRs opened regarding this change.

Reply:
in raw github urls shall i use "https://github.com/meshery/meshery-osm/tree/master/.github/welcome/Meshery-celebration-dark.svg"??

Reply:
Thanks @thisiskaransgit I overlooked this 😓 @Ghat0tkach  proceed as guided

Reply:
@Ghat0tkach 
https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/Meshery-celebration-dark.svg
https://raw.githubusercontent.com/meshery/meshery-osm/master/.github/welcome/Meshery-celebration-light.svg